docs/docs/en/guide/resource/configuration.md

```diff
@@ -11,7 +11,7 @@ For a single machine, you can choose to use local file directory as the upload d
 Configure the file in the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`.
 
 - Change `data.basedir.path` to the local directory path. Please make sure the user who deploy dolphinscheduler have read and write permissions, such as: `data.basedir.path=/tmp/dolphinscheduler`. And the directory you configured will be auto-created if it does not exists.
-- Modify the following two parameters, `resource.storage.type=HDFS` and `fs.defaultFS=file:///`.
+- Modify the following two parameters, `resource.storage.type=HDFS` and `fs.defaultFS=file:///` or modify `resource.storage.type=LOCAL` alone, it will use HDFS local mode and will automatically configure `fs.defaultFS=file:///`.
```

Comment:
We should suggest users using the `LOCAL` directly, and manually change can be backup
```suggestion
- modify `resource.storage.type=LOCAL` which will use HDFS local mode and will automatically configure `fs.defaultFS=file:///`. And you can also modify the following two parameters, `resource.storage.type=HDFS` and `fs.defaultFS=file:///` manually.
```

Reply:
Please also change the ZH doc